hl-alyx-sdk-setup.bat

```diff
@@ -58,18 +63,17 @@ ECHO :::::::::::::::::::::::::::::::::::::::
 ECHO Copying SteamVR files....
 ECHO :::::::::::::::::::::::::::::::::::::::
 robocopy "%steamvr_dir%/tools/steamvr_environments/game/bin" "%alyx_sdk_dir%/game/bin" /s /e /nfl /ndl /njh 
-robocopy "%steamvr_dir%/tools/steamvr_environments/game/core" "%alyx_sdk_dir%/game/core" /s /e /nfl /ndl /njh /XF pak* /XF shaders*
```

Comment:
Is copying the .pak files over the new SDK directory enough to use them? Now that they are being copied over and users are no longer being directed to extract them using GCFscape or similar tools, will the setup still function properly?

Reply:
yeah, vpk mounting works fine, including asset browser.